lib/seattleflu/api/static/index.html

```diff
@@ -74,8 +74,15 @@ <h3 class="code">POST /v1/receiving/consensus-genome</h3>
     <p>Stores the request data as a consensus genome document in a receiving
     area of the study database.
 
+    <h3 class="code">Get /v1/shipping/augur-build-metadata</h3>
+    <p>Exports consensus genome metadata necessay for SFS augur-build.
+
+    <h3 class="code">Get /v1/shipping/genomic-data/{lineage}/{segment}</h3>
```

Comment:
`Get`→`GET` to match standard HTTP method casing (e.g. as `POST` is in the endpoints above)